mgmtfn/dockplugin/netDriver.go

```diff
@@ -240,12 +241,24 @@ func createEndpoint(hostname string) func(http.ResponseWriter, *http.Request) {
 
 		log.Debug(ep)
 
+		// need to get the subnetlen from nw state.
+		nw, err := netdGetNetwork(netID)
+		if err != nil {
+			httpError(w, "Could not find created network", err)
+			return
+		}
+
 		epResponse := api.CreateEndpointResponse{
 			Interface: &api.EndpointInterface{
 				MacAddress: mresp.EndpointConfig.MacAddress,
 			},
 		}
 
+		// setup IPv6 address
+		if ep.IPv6Address != "" {
+			epResponse.Interface.AddressIPv6 = ep.IPv6Address + "/" + strconv.Itoa(int(nw.IPv6SubnetLen))
```

Comment:
@yuva29. Thanks for taking care of the IPv6 changes. 
For Docker, can you update the docker network creation API with EnableIPv6 flag instead of returning the IPv6 address in the endpoint create response.
https://github.com/contiv/netplugin/blob/master/netmaster/docknet/docknet.go#L180